pulpcore/pulpcore/app/models/task.py

```diff
@@ -117,11 +133,27 @@ class Worker(Model):
 
     name = models.TextField(db_index=True, unique=True)
     last_heartbeat = models.DateTimeField(auto_now=True)
-    online = models.BooleanField(default=True)
+    gracefully_stopped = models.BooleanField(default=True)
 
-    def save_heartbeat(self):
-        """Save a worker heartbeat
+    @property
+    def online(self):
+        """
+        Whether a worker can be considered 'online'
+
+        To be considered 'online', a worker must have a recent heartbeat timestamp and must not
+        have the 'gracefully_stopped' flag set to True. "Recent" is defined here as "within
+        the pulp process timeout interval".
+
+        Returns:
+            bool: Whether the worker is online or not.
```

Comment:
Would this be clearer if it was written `True if the worker is online, False otherwise`? Just an idea.